rj_gameplay/rj_gameplay/tactic/wall_tactic.py

```diff
@@ -45,6 +45,13 @@ def __call__(self, robot: rc.Robot, prev_result: Optional["RoleResult"],
         return np.linalg.norm(robot.pose[0:2] - self.wall_pt
                               ) / global_parameters.soccer.robot.max_speed
 
+    def unassigned_cost_fn(
```

Comment:
For this and other non required cost functions I don't think the correct behavior for this is a big number like this.

Reply:
Yup, just a placeholder since they are now required for role assignment, will put in TODOs.